content/events/2020/10/2020-10-20-customer-experience-continuous-improvement-the-usps-digital-approach.md

```diff
@@ -0,0 +1,64 @@
+
+The USPS has adopted a Customer Experience (CX) continuous improvement approach to blend federal digital standards and best practices with the tools and techniques of private industry and eCommerce. Our method merges analytics and customer feedback with iterative heuristic reviews, design, and usability testing. This approach allows us to monitor, improve, and evolve our CX from both the quantitative and qualitative perspectives. 
+
+Attendees will walk away with an understanding of:
+
+* The growth and evolution of the USPS.com analytics and feedback practice
+* An approach to combining qualitative customer survey data and quantitative website traffic data
+* How USPS has used targeted, reoccurring and ad-hoc reports that promote a data-driven, evidence-based design approach while focusing usability studies
+* An introduction to the USPS’ customized usability heuristics (mapped to the Usability.gov Research-Based Web Design Usability Guidelines (WDUG)), team-based heuristic evaluation approach, and menu of usability testing services
+* The importance of engaging in continuous, often iterative heuristic evaluations, compliance reviews, and usability testing
+* Methods of benchmarking against peers, competitors, and best-in-class eCommerce sites to remain current with the evolving digital landscape and expectations
+* How the above analytics and continuous improvement approach has helped USPS promote, protect, and expand its brand while growing revenue (especially in extraordinary times, like during the novel coronavirus pandemic).
+
+## Presenters:
+
+**Matt Petrowski** began his digital career in 2007 as an ad ops trafficker for HealthCentral, a top online health publisher. After subsequent work as a digital strategist and eCommerce content manager, Matt joined the USPS digital brand marketing team in 2014 as the digital analytics program manager. Since then, he has successfully transitioned USPS.com to Google Analytics, incorporated structured data on a platform level, implemented a centralized digital marketing data platform, and shared machine learning from website data with USPS business teams. 
+
+Outside of work, Matt’s primary area of interest is how digital mediums modify interpersonal behaviors and emotions. 
+
+**Jeff Pass** made his first website in 1997 and has been involved in digital communications, usability, information architecture, content strategy, and design ever since. For the last 21 years, Jeff has been a federal contractor working on high-profile projects including HealthCare.gov, CFTC.gov, MyHealthevet.gov (now MyHealth.va.gov), and dozens more. Since 2014, he has been the User Experience Team Lead for USPS.gov.
+
+Jeff also helps the UX community of practice for his employer, Booz Allen Hamilton, and has been a long-time UX event organizer, including GoodGov UX, World IA Day DC, UXDC, DCUS, World Usability Day, IA Summit, IA Conference, and UXPA.
+
+---
+
+  **Related Links:**
```

Comment:
I don't think we should have the text linked and then also the individual links. It would be quite repetitive for screen readers. @ToniBonittoGSA What is the best practice here for accessibility?